Inseok_45/services/likeService.js

```diff
@@ -0,0 +1,26 @@
+const likeDao = require('../models/likeDao');
+
+const likePosts = async ( userId, postId ) => {
+    try {
+        const result = await likeDao.checkLike(userId, postId);
+
+        if(result === 0) {
```

Comment:
```suggestion
        if(!result) {
```
똑같이 동작하면서 코드의 흐름파악이 더 수월합니다 :)